pre_commit/languages/python.py

```diff
@@ -57,6 +58,9 @@ def install_environment(repo_cmd_runner, version='default'):
         repo_cmd_runner.run(venv_cmd)
         with in_env(repo_cmd_runner, version) as env:
             env.run("cd '{prefix}' && pip install .")
+            if additional_dependencies:
+                env.run("cd '{prefix}' && pip install -U" +
```

Comment:
I wouldn't use `--upgrade`, it has a tendency to cause incompatibilities with the libraries installed.  I think `pip install ...` should be sufficient